index.html

```diff
@@ -218,52 +218,23 @@ <h3 class="name-project">project name goes here</h3>
               <li class="lang-project">Ruby on rails</li>
               <li class="lang-project">Javascript</li>
             </ul>
-            <button class="button-project">
+            <a class="button-project" id="btn6-1" href="#header12">
               See this project
               <span
                 ><img src="./assests/images/Union.png" alt="arrow image"
               /></span>
-            </button>
-            <button class="button-project768">
+            </a>
+            <a class="button-project768" id="btn6-2" href="#header12">
               See this project
               <span
                 ><img src="./assests/images/Union2.png" alt="arrow image"
               /></span>
-            </button>
+            </a>
           </article>
         </article>
-        <article class="pop-project1">
-          <div class="pop-header"><button class="pop-close"><img src="./assests/images/Union.svg" alt=""></button></div>
-          <h3 class="pop-heading1">project name goes here</h3>
-          <ul class="pop-ulist1">
-            <li class="pop-li1">HTML/CSS</li>
-            <li class="pop-li1">Ruby on rails</li>
-            <li class="pop-li1">Javascript</li>
-          </ul>
-          <div class="pop-image1">
-            <img src="./assests/images/Rectangle21.png" alt="project-photo" class="pop-big"/>
-            <img src="./assests/images/Rectangle21.png" alt="project-photo" class="pop-small"/>
-            <img src="./assests/images/Rectangle21.png" alt="project-photo" class="pop-small"/>
-            <img src="./assests/images/Rectangle21.png" alt="project-photo" class="pop-small"/>
-            <img src="./assests/images/Rectangle21.png" alt="project-photo" class="pop-small"/>
-          </div>
-          <p class="pop-paragraph1">
-            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Itaque
-            repudiandae tempora nisi minima non beatae aspernatur praesentium
-            ipsa? Quia quidem debitis error tenetur, ea eos facere natus tempore
-            neque ducimus doloribus assumenda aspernatur iusto soluta enim at
-            accusantium impedit ratione quasi perferendis cum hic esse ipsum
-            sit!<br>br Atque corrupti unde impedit commodi voluptate cum inventore
-            tenetur sequi accusantium fugiat consectetur provident voluptatem
-            blanditiis quae excepturi repudiandae sunt rem ipsam, magni voluptas
-            ullam ab nulla vero exercitationem! Tempora, consequatur velit! Hic
-            aut explicabo voluptates sed ipsum eos praesentium a inventore fuga
-            mollitia nostrum, ducimus incidunt! Quos quo in rerum blanditiis
-            perferendis!
-          </p>
-          <button class="pop-live1"></button>
-          <button class="pop-source1"></button>
-        </article>
+      </section>
+      <section class="popup-window">
+        <article class="pop-container" id="cont-pop"></article>
       </section>
```

Comment:
- Hi @basem909 sorry 🙏 we missed this part of the project requirement that says `all of the HTML in that section is created when the page loads.` see screenshot below ⬇️ 
![image](https://user-images.githubusercontent.com/52497246/162546065-bd71f546-706b-4fee-bb14-84247fc9f0ec.png)

- Kindly take out the HTML for the project section both the project and popup action should be done dynamically so that on page load the project cards would be displayed and when the `See Project ` button is clicked the popup also is rendered  🙏